packages/react-component/src/main.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect, useRef } from 'react'
+import { embed } from "@genome-spy/core";
+import type { ViewSpecBase, RootSpec } from "@genome-spy/core"
+import { EmbedResult } from '@genome-spy/core/types/embedApi.js';
+
+interface IGenomeSpyProps {
+    spec: ViewSpecBase;
+    onEmbed: (api: Promise<EmbedResult>) => void; 
+}
+
+export default function GenomeSpy ({ spec, onEmbed }: IGenomeSpyProps) {
+    const embedRef = useRef<HTMLDivElement>(null)
+    const [error, setError] = useState<string | undefined>()
+
+    useEffect(() => {
+        async function embedToDoc(container: HTMLDivElement | null, config: RootSpec) {
```

Comment:
Not embedding to docs. Function name should be changed.

Reply:
Thanks, should be in 093b914